assets/_scss/core/_grid.scss

```diff
@@ -99,9 +99,17 @@
     }
   }
 
+  // Specificies end of a row.
+  // Required if grid-box contains multiple rows.
+  // Required if browser does not support :last-child
   .usa-end-row {
     @include omega();
   }
+
+  // Fixes grid alignment when grid-items have varying heights.
+  .usa-end-row + .grid-item {
```

Comment:
Actually I think what it is is we're not using the grid-item class. Is that right?